framework/tlvf/src/include/tlvf/CmduMessageRx.h

```diff
@@ -22,14 +22,14 @@ class CmduParser;
 class CmduMessageRx : public CmduMessage {
 
 public:
-    CmduMessageRx();
-    CmduMessageRx(CmduMessageRx &original);
+    CmduMessageRx() = delete;
+    CmduMessageRx(uint8_t *buff, size_t buff_len);
     ~CmduMessageRx();
-    bool parse(uint8_t *buff, size_t buff_len, bool swap_needed = true, bool parse_tlvs = false);
+    bool parse(bool swap_needed = true, bool parse_tlvs = false);
     CmduMessageRx &operator=(const CmduMessageRx &) = delete;
 
 private:
-    std::list<std::shared_ptr<CmduParser>> parsers_;
+    std::shared_ptr<CmduParser> parser;
```

Comment:
Yes, but at this point what is there to gain from it? I prefer not to block merge for this PR for small things like this one since every new change that enters prplMesh may break this one (we'll need to find all those places we need to replace `cmdu_tx.addClass()` with `beerocks_header->addClass()`)